src/ch.usi.inf.nodeprof.test/js/minitests/fib2.js

```diff
@@ -0,0 +1,7 @@
+function fib(n){
+    if (n < 2)
+        return n;
+    return fib(n-1)+fib(n-2);
+}
+
+fib(1);
```

Comment:
There is no return event here for fib(1), to be investigated.

Reply:
any updates on this? Is this a Graal.js problem? If so, could you investigate? If you have time, please go ahead and open a PR on the Graal.js github repo with a fix (or ping me on slack if you have troubles fixing this)

Reply:
It's a bug at my part. It should be fixed now. I didn't pay attention to your test code and was using the node return value as the value being returned, which is not the case for FrameReturnNode. Now I change it to use the first input and it works now.